### DISCUSSION TITLE
Render Data fetched from External Server in Tanstack Table

### DISCUSSION THREAD

**Discussion by nvijayap** [TanStack/table#5169] [Q&A]
Hi,

I have this working piece of code in my nuxt3 app ...

```
<template>
  <div>
    <p v-for="user in users" :key="user.id">{{ user.name }}</p>
  </div>
</template>

<script>
export default {
  data () {
    return {
      users: {},
    }
  },
  async mounted() {
    this.users = await $fetch('https://jsonplaceholder.typicode.com/users')
  },
}
</script>
```

How can I refactor it to show the data in Tanstack Table?

Thanks

**Comment by Gabswim** [comment]
Hi,
There is an example, but in React: [React Example: Pagination Controlled](https://tanstack.com/table/v8/docs/examples/react/pagination-controlled). Let me know if that helps, if not I can help you further with what you want to accomplish.

**Reply by nvijayap** [reply]
Hi,

Thanks, but that is for `React`, I am looking for one in `Nuxt3`

Thanks
